indexer/Makefile

```diff
@@ -7,6 +7,8 @@ LDFLAGSSTRING +=-X main.GitDate=$(GITDATE)
 LDFLAGSSTRING +=-X main.GitVersion=$(GITVERSION)
 LDFLAGS := -ldflags "$(LDFLAGSSTRING)"
 
+SCC_ABI_ARTIFACT = ../packages/contracts/artifacts/contracts/L1/rollup/StateCommitmentChain.sol/StateCommitmentChain.json
```

Comment:
is this file checked in?

Reply:
No it's not - it's an artifact generated during compilation of the legacy contracts.

Reply:
Ok, can you leave a comment in the makefile to that effect? That means that running this make command on a fresh repo would fail if they haven't already been built. Also, do we ever intend to change these?

Reply:
Unlikely they will ever change, we used this pattern in the past. There will come a time when we delete the `contracts` package and rename `contracts-bedrock` to `contracts`. I wonder if a solidity interface can be used to generate bindings, if so we could add the scc interface to `contracts-bedrock` and generate the bindings that way but then we won't have the deployment functionality, which i suppose we need for testing